package.json

```diff
@@ -158,10 +158,10 @@
 					"default": false,
 					"description": "Enable (experimental) code lens for function definitions."
 				},
-				"rescript.settings.signatureHelp.enable": {
+				"rescript.settings.signatureHelp.enabled": {
```

Comment:
Changing the name of this setting so it resets for all users, and defaults to true.